react/ConditionLayoutProduct.tsx

```diff
@@ -28,6 +28,7 @@ export const PRODUCT_SUBJECTS = {
 
 const Product: StorefrontFunctionComponent = ({ children }) => {
   const { product, selectedItem } = useProduct() as any
+  if (!product) return null
```

Comment:
I don't know if the right behaviour should be to:
1. Render nothing
2. Let it render the child blocks, making `condition.else` to render

Reply:
I think I'd render the else block as well. In what occasion can `product` be null?

Reply:
Sometimes it takes a while for this information to come @kaisermann

Reply:
Besides that, I don't know if there is any scenario in which this block can be used but we don't have access to the product

Reply:
Hm.... in that case, I don't think we should render the `else` block. Better to show nothing than to show something incorrect, because the eventual product to be loaded could trigger one of the other conditions at some point in the future.